apps/journeys-admin/src/components/Editor/Slider/JourneyFlow/nodes/StepBlockNode/StepBlockNodeMenu/StepBlockNodeMenu.stories.tsx

```diff
@@ -0,0 +1,61 @@
+import { MockedProvider, MockedResponse } from '@apollo/client/testing'
+import Box from '@mui/material/Box'
+import { expect } from '@storybook/jest'
+import { Meta, StoryObj } from '@storybook/react'
+import { screen, userEvent, waitFor } from '@storybook/testing-library'
+import { ComponentProps } from 'react'
+
+import type { TreeBlock } from '@core/journeys/ui/block'
+
+import { BlockFields_StepBlock as StepBlock } from '../../../../../../../../__generated__/BlockFields'
+import { journeysAdminConfig } from '../../../../../../../libs/storybook'
+import { ThemeProvider } from '../../../../../../ThemeProvider'
+
+import { StepBlockNodeMenu } from './StepBlockNodeMenu'
+
+const Demo: Meta<typeof StepBlockNodeMenu> = {
+  ...journeysAdminConfig,
+  component: StepBlockNodeMenu,
+  title:
+    'Journeys-Admin/Editor/Slider/JourneyFlow/nodes/StepBlockNode/StepBlockNodeMenu'
+}
+
+const step: TreeBlock<StepBlock> = {
+  id: 'step1.id',
+  __typename: 'StepBlock',
+  parentBlockId: null,
+  nextBlockId: null,
+  parentOrder: 0,
+  locked: false,
+  children: []
+}
+
+const Template: StoryObj<
+  ComponentProps<typeof StepBlockNodeMenu> & { mocks: MockedResponse[] }
```

Comment:
You don't need to extend the type here since you're not passing any mocks in
```suggestion
  ComponentProps<typeof StepBlockNodeMenu>
```